robolectric/src/test/java/org/robolectric/shadows/ShadowCanvasTest.java

```diff
@@ -529,4 +530,19 @@ public void restoreToCount() {
 
     assertThat(canvas.getSaveCount()).isEqualTo(save1);
   }
+
```

Comment:
We also need a test for `Canvas#drawRect(RectF rect, Paint paint)`.

Reply:
It's also recommended to add test to cover corner case pointed out by @hoisie at https://github.com/robolectric/robolectric/pull/7197#discussion_r839174063.

Reply:
I would probably recommend not worrying about this corner case (e.g. painting with invisible colors, e.g. 0x00ffffff). It's an edge case that is okay if it's not supported. We can look into it later if necessary.

Reply:
@utzcoz Is it for `Canvas#drawRect(RectF rect, Paint paint)` or `Canvas#drawRect(Rect rect, Paint paint)`..

Reply:
I think it should be for `Canvas#drawRect(Rect rect, Paint paint)`.

Reply:
@Akshay2131 I think it is for `Canvas#drawRect(RectF rect, Paint paint)`(https://developer.android.com/reference/android/graphics/Canvas#drawRect(android.graphics.RectF,%20android.graphics.Paint)). If you found Robolectric missed tests for other variants of `Canvas#drawRect`, it's recommended to add tests for them.